docs/reference/data-streams/lifecycle/index.asciidoc

```diff
@@ -16,7 +16,8 @@ To achieve that, it supports:
 * Automatic <<index-rollover,rollover>>, which chunks your incoming data in smaller pieces to facilitate better performance
 and backwards incompatible mapping changes.
 * Configurable retention, which allows you to configure the time period for which your data is guaranteed to be stored.
-{es} is allowed at a later time to delete data older than this time period.
+{es} is allowed at a later time to delete data older than this time period. Retention can be configured on data stream level
```

Comment:
`can be configured on data stream level` -> `can be configured on the data stream level` maybe?
Also this is kind of a run-on sentence. It would probably be good to break it into two sentences, the second one starting "Read more about..."